index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header class="container">
+        <div class="logo">My Logo</div>
+
+        <nav class="menu-icon">
+            <img src="./images/ic_menu.jpg" alt="">
+        </nav>
+
+        <div class="circle">
+            <br id="rec">
+
+        </div>
+
+
+        <section class="headline">
+            <div class="intro">
+                <h1>i'm Adan Glad to see you!</h1>
+            </div>
+            <div class="sumarry">
+                <p class="about-me">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.</p>
+            </div>
```

Comment:
You are so close to the original design idea, but all you need to do is to follow the exact font and text size. Please make sure to update your font, and text size in here.